tools/certissuer/lego.go

```
@@ -0,0 +1,134 @@
+	"github.com/go-acme/lego/v4/providers/dns/cloudflare"
+	"github.com/go-acme/lego/v4/providers/dns/gcloud"
+	"github.com/go-acme/lego/v4/registration"
+	"github.com/hashicorp/go-retryablehttp"
+	"github.com/tendermint/tendermint/libs/log"
+)
+
+type CertIssuer interface {
+}
+
+type certIssuer struct {
+	ctx context.Context
```

golangci-lint (1.64.8)

field `ctx` is unused

(unused)